product_pack_category_restriction/models/product_category.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductCategory(models.Model):
+    _inherit = "product.category"
+
+    pack_ok = fields.Boolean(string="Is Pack Category?")
```

Comment:
The default should be compatible.

```suggestion
    pack_ok = fields.Boolean(string="Is Pack Category?", default=True)
```